modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/resources/META-INF/resources/js/SystemDataSets.tsx

```diff
@@ -58,14 +161,16 @@ const SystemDataSets = ({namespace}: {namespace: string}) => {
 		<div className="data-sets system-data-sets">
 			<FrontendDataSet
 				{...FDS_DEFAULT_PROPS}
+				apiURL={API_URL.DATA_SETS}
```

Comment:
Not a matter of this task but we'll need to filter this to show the customizations only. Likewise with custom data sets `apiURL`